specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/examples/RandomNumbers-example.json

```diff
@@ -3,7 +3,7 @@
     "vaultBaseUrl": "https://myhsm.managedhsm.azure.net/",
     "api-version": "7.3-preview",
     "parameters": {
-      "bytesLength": 4
+      "bytes_length": 4
```

Comment:
Is this what the service actually uses? This is not typical. JSON schema for RESTful methods should use camelCase. I'm not even sure if the SDK generators will handle this correctly.

Reply:
Yes, "bytes_length" is the name that the service is using. But no problem, I will change it back to "bytesLength" in this repo if this is the format that the SDK generators handle.

Reply:
Does that also mean you'll change the service to match then?

Reply:
As far as I know, the variable name in the service doesn't need to match the name in JSON/swagger specification. In the service code, variable name is "bytes_length" and the json property name I use is "bytesLength" (to match the swagger specification). So no worries.

Reply:
Model names don't need to match (model names aren't present in JSON payloads), but properties do. If you're expecting/parsing a property named "bytes_length", you won't parse "bytesLength".

I recommend changing the service to match all other properties that are camelCase today while the feature is new.

Reply:
Yes, I agree. That's what I meant (Sorry if my previous message caused confusion). The JSON property name that I use/expect to parse in the service is "bytesLength" , which matches the name in this repo.